src/lxml/xmlerror.pxi

```diff
@@ -73,6 +76,7 @@ cdef class _LogEntry:
         self.column   = error.int2
         self._c_message = NULL
         self._c_filename = NULL
+        self._c_path = NULL
```

Comment:
I had a question on the mailing list about this. By my reading of [the Cython documentation](http://cython.readthedocs.io/en/latest/src/userguide/special_methods.html#initialisation-methods-cinit-and-init), this initialization to NULL has already been taken care of (same for the previous two assignments, as well as the one of `self._c_path` in `_setGeneric()` below). Is this correct? Can these initializations have been safely omitted? I'm not saying we should necessarily take them out. I'm just trying to solidify my understanding of the workings of Cython.

Thanks!